solutions/security/get-started/spaces-security-faq.md

```diff
@@ -0,0 +1,234 @@
+---
+applies_to:
+  stack: preview 9.1
+  serverless:
+    security: all
+products:
+  - id: security
+  - id: cloud-serverless
+---
+
+# Spaces and {{elastic-sec}} FAQ [security-spaces-faq]
```

Comment:
I think the title of this page (and throughout the content below) should be specific for Elastic Defend, since these are only applicable to that component from within Elastic Security. They do not apply to how spaces work for other Security areas (ex. SIEM, Timeline, etc)